selfdrive/car/honda/interface.py

```diff
@@ -200,6 +200,19 @@ def get_params(candidate, fingerprint):
       ret.sR = 15.3
       ret.steerKp, ret.steerKi = 0.8, 0.24
 
+      ret.longitudinalKpBP = [0., 5., 35.]
+      ret.longitudinalKpV = [1.2, 0.8, 0.5]
+      ret.longitudinalKiBP = [0., 35.]
+      ret.longitudinalKiV = [0.18, 0.12]
+    elif candidate == "HONDA ODYSSEY 2018 EX-L":
+      stop_and_go = False
+      ret.m = 4354./2.205 + std_cargo
+      ret.l = 3.00
+      # Wtf are these?
```

Comment:
Allright, I'll change those var names into more explicit ones. 
Definitions are in cereal/car.capnp

Reply:
Long names!

Reply:
fixed. @energee, will merge in openpilot devel soon.